### ISSUE TITLE
set scatter parameters according to Hogg's heuristics

### ISSUE THREAD

**Issue by davidwhogg** [andycasey/AnniesLasso#31]
...Hogg supply details here

**Comment by davidwhogg** [comment]
The heuristic is:

- At each wavelength j, find the value s_j^2 that makes the mean training-set chi value 1.
- chi is defined to be the residual (in flux space) between the best-fit model and the observed datum, divided by the standard error (which has s_j added in quadrature).

Once s_j^2 values are chosen, re-optimize with those s_j^2 fixed.

**Comment by andycasey** [comment]
Somehow I missed this second comment that actually supplied the heuristic.

I think this is good. But up until now we have been using `s2 = 0`, so I should set the scatter for our chosen model using this heuristic and re-run it?

**Comment by davidwhogg** [comment]
Yes, let's try that.  And let me know if x-validation gets better or worse.

**Comment by andycasey** [comment]
Better. This is using our regularized model (`Lambda = 10^3`, `f = 2.0`, `s^2` set by your heuristic) on individual spectra in the validation set. 

This figure is now in the paper with accompanying text. The functionality for this issue is all in the code.

So, I'm closing this issue.

![image](https://cloud.githubusercontent.com/assets/504436/13086288/8650b442-d4da-11e5-9817-6aba5a674882.png)
